frontend/src/components/MapView/AlertForm/index.test.tsx

```diff
@@ -3,10 +3,8 @@ import { Provider } from 'react-redux';
 import { render } from '@testing-library/react';
 import { ThemeProvider } from '@material-ui/styles';
 import { createMuiTheme } from '@material-ui/core/styles';
+import { store } from 'context/store';
 import AlertForm from '.';
-import { store } from '../../../context/store';
-
-jest.mock('../Layers/LayerDropdown', () => 'mock-Layer-Dropdown');
```

Comment:
@HarryMytilinaios that's the reason for the snapshot changes

Reply:
Yes I re-added it. I was trying to find a way for jest to identify the absolute paths, but everything I did in a `jest.config.js` was not correct, and I wasn't sure how to make jest to identify the mock modules through absolute and not relative paths, this is the reason that this was mistakenly deleted. Right now absolute paths are working for every file module but the `jest.mock` is only working with relative paths